tooling/build/scripts/publishing/index.ts

```diff
@@ -80,6 +86,7 @@ async function main() {
       if (
         (resource.type === "Page" ||
           resource.type === "CollectionPage" ||
+          resource.type === "IndexPage" ||
```

Comment:
do we need to accuont for `CollectionLink`? i remember we rendered an empty layout but double checking here

Reply:
Yup we wil need to, but right now cos the `CollectionLink` type is still in your PR (sad) so I can't type it here.